tests/test_main.py

```diff
@@ -1,11 +1,125 @@
+import uuid
+import pytest
 from fastapi.testclient import TestClient
-
 from src.main import app
+from typing import Generator
+
+from src.schemas import Project, ProjectDetails
+
+
+@pytest.fixture
+def client() -> Generator[TestClient, None, None]:
```

Comment:
This is correct, also from python 3.13 there is a huge QoL improvement - now generics have default values including Generator, meaning you can do something neat as this:
```suggestion
def client() -> Generator[TestClient]:
```
Pep regarding this: https://peps.python.org/pep-0696/
Video regarding this: https://www.youtube.com/watch?v=NC3Bv104SQM